src/gui/calibration_ui.py

```
@@ -0,0 +1,371 @@
+            self.root.attributes('-topmost', True)
+        
+        # 背景色設定
+        self.root.configure(bg='gray')
+        
+    def _get_scale_factor(self) -> float:
+        """ディスプレイのスケールファクターを取得"""
+        # Retinaディスプレイ対応
+        try:
+            # tkinterから取得を試みる
+            return self.root.tk.call('tk', 'scaling')
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)